### ISSUE TITLE
[FEATURE REQUEST] Toggle filter for scene

### ISSUE THREAD

**Issue by bonaccolto90** [rivafarabi/deckboard#615]
Is it possible to add a Toggle Filter for scenes on "OBS Studio" section? Saw on the software that there is a toggle filter, but for the sources on a scene...
I want to use it to change the position of my webcam through a plugin installed on OBS that use filters to make this possible

**Comment by tobim91** [comment]
Yeah want this feature too because saw a tutorial where you can move your webcam in different corners.

**Comment by TheRed-E** [comment]
Would be a greate feature! Also wanting to move stuff around in the scean by the press of a button. Due to missing this option, it is currenly not posible.

**Comment by bonaccolto90** [comment]
I manage to "fix" the lack of this option by using the F13-F24 trick (also could it be done with NOOBS CMDR, made by [Nutty](https://github.com/nuttylmao/NOOBS-CMDR)... you can check this [tutorial](https://www.youtube.com/watch?v=1x3QqZHr6IY))

**Comment by TheRed-E** [comment]
Found it! Got it working in a few seconds that way! GREAT!

**Comment by miguelfabio** [comment]
For those with this issue, I found a workaround that you won't need any app else.

First, in OBS, create a new scene and add a source scene to it. Then, select the scene where you added the filters.

![image](https://user-images.githubusercontent.com/119859063/206053784-2a62d7a1-9365-4207-92dc-1137ae8842b9.png)

Now, on Deckboard, create a new button, select "Toggle Filter", select the new scene you just created and in the "source" list select the scene with the filters. Now you'll be able to select the filter from the scene.

![image](https://user-images.githubusercontent.com/119859063/206054282-eec34d58-5f46-45ba-ad01-fcdf66bdb2f1.png)

Hit "ADD" and you're good to go. You can create other buttons to select all the filters you need. It should work without having to worry about other apps.

![image](https://user-images.githubusercontent.com/119859063/206054957-acbf4a63-102a-4569-a36f-e9fb418322e3.png)

Hope this can help others, and hope this feature can be implemented to Deckboard to natively support the Scene Toggle Filter.
